service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/DeleteAzureDatabaseStepUnitTest.java

```diff
@@ -0,0 +1,235 @@
+package bio.terra.workspace.service.resource.controlled.cloud.azure.database;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.*;
+
+import bio.terra.common.iam.BearerToken;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.workspace.amalgam.landingzone.azure.LandingZoneApiDispatch;
+import bio.terra.workspace.app.configuration.external.AzureConfiguration;
+import bio.terra.workspace.common.exception.AzureManagementExceptionUtils;
+import bio.terra.workspace.common.utils.BaseMockitoStrictStubbingTest;
+import bio.terra.workspace.generated.model.ApiAzureLandingZoneDeployedResource;
+import bio.terra.workspace.service.crl.CrlService;
+import bio.terra.workspace.service.iam.SamService;
+import bio.terra.workspace.service.workspace.WorkspaceService;
+import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys;
+import bio.terra.workspace.service.workspace.model.AzureCloudContext;
+import bio.terra.workspace.service.workspace.model.Workspace;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.management.exception.ManagementError;
+import com.azure.core.management.exception.ManagementException;
+import com.azure.resourcemanager.postgresqlflexibleserver.PostgreSqlManager;
+import com.azure.resourcemanager.postgresqlflexibleserver.models.Databases;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+@Tag("unit")
+public class DeleteAzureDatabaseStepUnitTest extends BaseMockitoStrictStubbingTest {
+
+  @Mock AzureConfiguration azureConfig;
+  @Mock CrlService crlService;
+  @Mock SamService samService;
+  @Mock WorkspaceService workspaceService;
+  @Mock LandingZoneApiDispatch landingZoneApiDispatch;
+  @Mock ControlledAzureDatabaseResource resource;
+  @Mock PostgreSqlManager postgresManager;
+  @Mock Databases databases;
+
+  @Mock AzureCloudContext azureCloudContext;
+  @Mock FlightMap workingMap;
+  @Mock FlightContext context;
+
+  @BeforeEach
+  void localSetup() {
+    Mockito.lenient().when(context.getWorkingMap()).thenReturn(workingMap);
```

Comment:
All of these need to be lenient?

Reply:
Not anymore, apparently!
I think it was a hold-over from when I still had these tests mixed in with some other spring-boot tests.